src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -74,6 +74,8 @@
     strategy.MUTATOR_PLUGIN_RADAMSA_STRATEGY.name
 ]
 
+DISABLE_SYMLINK_SANITIZER_VAR = 'SYSTEMSAN_MALICIOUS_SYMLINK_FOLLOWING'
```

Comment:
How is this env var used?

Reply:
+1. Is this needed?

Reply:
I pushed a change to SystemSan to allow selective disabling https://github.com/google/oss-fuzz/pull/9221

Reply:
Ah I see, it breaks cleanse. Maybe we should just be disabling systemsan completely instead for anything that's not reproducing or fuzzing?

Reply:
Yeah this is what I should be doing

Reply:
libFuzzer is vulnerable to this issue btw https://source.chromium.org/chromium/chromium/src/+/main:third_party/libFuzzer/src/FuzzerIO.cpp;l=156?q=tmpdir&ss=chromium%2Fchromium%2Fsrc:third_party%2FlibFuzzer%2Fsrc%2F that's why it breaks. Maybe this issue is too unimportant to report. WDYT?

Reply:
Let's rename this var to "ENABLE_SYMLINK_SANITIZER_VAR" instead since a 0 means disable and 1 means enable.